lib/webdriveragent.js

```diff
@@ -59,6 +61,10 @@ class WebDriverAgent {
 
     this.updatedWDABundleId = args.updatedWDABundleId;
 
+    this.usePreinstalledWDA = args.usePreinstalledWDA;
+    this.preInstalledWDABundleId = args.preInstalledWDABundleId;
+    this.xctestWDA = null;
```

Comment:
I think we could have better name for this property

Reply:
What about `xctestSessionForWDA`?